WalletWasabi.Fluent.Desktop/Program.cs

```diff
@@ -283,3 +138,60 @@ private static AppBuilder BuildCrashReporterApp(SerializableException serializab
 			.AfterSetup(_ => ThemeHelper.ApplyTheme(Theme.Dark));
 	}
 }
+
+public static class WasabiAppExtensions
+{
+	public static async Task<int> RunAsGUIAsync(this WasabiApplication app)
+	{
+		return await app.RunAsync(
```

Comment:
While I heard that this pattern is used to minimize possible errors caused by `async`, I cannot resist from leaving a comment. When a method awaits a task and does nothing else that task can be returned immediately. The cost of one useless state machine removal is that there will be no `RunAsGUIAsync` in the stack trace.

Another nit is that according to .NET naming conventions (which are not always respected) the method should be named as `RunAsGuiAsync`.